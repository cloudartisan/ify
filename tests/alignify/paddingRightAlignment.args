-i = -p 1 -r
