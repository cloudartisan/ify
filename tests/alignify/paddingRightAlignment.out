abc  = 1
def  = 2
gh   = 3
